### PR TITLE
Add section about code conventions to DEVELOP.md

### PR DESCRIPTION
**Pull request by jascott1**
This adds details about shfmt and shellcheck settings.

Signed-off-by: Justin Scott <justin.a.scott@intel.com>

### CODE REVIEW

File: clr-k8s-examples/DEVELOP.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
 # Developer
 
-This document describes the key concepts and technologies used in the project.
+This document describes the key concepts and technologies used in the project and how to contribute.
```
Comment (knangia) [inline clr-k8s-examples/DEVELOP.md]: Just listing few Grammatical nits, feel free to edit as you like:
This document describes the key concepts and technologies used in the project, and lists the ways to contribute to the project.

File: clr-k8s-examples/DEVELOP.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+
+## Code Conventions
+
+### Shell Scripts 
+
+Shell scripts should adhere to the [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml) as much as possible.
+
+#### Formatting with `shfmt`
+
+The [shfmt](https://github.com/mvdan/sh#shfmt) tool should be used to format shell scripts with 2 spaces and uses the following parameters:
```
Comment (knangia) [inline clr-k8s-examples/DEVELOP.md]: The [shfmt](https://github.com/mvdan/sh#shfmt) tool should be used to format shell scripts with 2 spaces and use the following parameters:

File: clr-k8s-examples/DEVELOP.md [hunk 3]
```diff
@@ -1,6 +1,33 @@
...
+
+```shell script
+shfmt -i 2 -ci
+``` 
+
+#### Linting with shellcheck
+
+The [shellcheck](https://github.com/koalaman/shellcheck) tool should be used to identify issues with the scripts 
+themselves. The config file for shellcheck is typically found in `~/.shellcheckrc` and should contain any rules 
+that are [ignored](https://github.com/koalaman/shellcheck/wiki/Ignore) project wide.
```
Comment (knangia) [inline clr-k8s-examples/DEVELOP.md]: and should include rules 
that are [ignored](https://github.com/koalaman/shellcheck/wiki/Ignore) project wide.

File: clr-k8s-examples/DEVELOP.md (hunk 1 above)
Comment (jascott1) [reply clr-k8s-examples/DEVELOP.md]: Fixed!

File: clr-k8s-examples/DEVELOP.md (hunk 2 above)
Comment (jascott1) [reply clr-k8s-examples/DEVELOP.md]: went with 'should use'. thanks!

File: clr-k8s-examples/DEVELOP.md (hunk 3 above)
Comment (jascott1) [reply clr-k8s-examples/DEVELOP.md]: Fixed!

**Pull request merged**